terratorch/cli_tools.py

```diff
@@ -352,13 +366,21 @@ def instantiate_classes(self) -> None:
             config = self.config
         if hasattr(config, "predict_output_dir"):
             self.trainer.predict_output_dir = config.predict_output_dir
-        
+
         if hasattr(config, "out_dtype"):
             self.trainer.out_dtype = config.out_dtype
 
         if hasattr(config, "deploy_config_file"):
             self.trainer.deploy_config = config.deploy_config_file
 
+        # Custom modules path
+        if hasattr(self.config.fit, "custom_modules_path"):
+
+            custom_modules_path =  self.config.fit.custom_modules_path
+        else:
+            custom_modules_path = os.environ.get("CUSTOM_MODULES_DIR_NAME", "custom_modules")
```

Comment:
maybe set the default to be an actual relative path. Given the point I raised above.
So I would do something like:
```
default_path = Path(".") / "custom_modules"
custom_modules_path = os.environ.get("TERRATORCH_CUSTOM_MODULE_PATH", default_path)
```